src/workloads/sharding/WriteOneWithoutShardKeyUnshardedCollection.yml

```diff
@@ -0,0 +1,119 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  On an unsharded collection on a single shard cluster, run workloads that updateOne, deleteOne, and findAndModifyOne.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+
+  Database: &Database test
+  UnshardedCollection: &UnshardedCollection Collection1
+
+  DocumentCount: &DocumentCount 10000  # Number of documents to insert and modify.
+  BalancerWait: &BalancerWait "20 seconds"  # Wait out the 20 second delay between balancing rounds
+  # with a little extra leeway for balancing to finish.
+
+Actors:
+- Name: CreateShardedCluster
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationMetricsName: EnableSharding
+      OperationName: AdminCommand
+      OperationCommand:
+        enableSharding: *Database
+    - OperationMetricsName: BalancerStart
+      OperationName: AdminCommand
+      OperationCommand:
+        balancerStart: 1
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: LoadInitialDataOnUnshardedCollection
+  Type: MonotonicSingleLoader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    SleepBefore: *BalancerWait
+    BatchSize: 1000
+    DocumentCount: *DocumentCount
+    Database: *Database
+    Collection: *UnshardedCollection
+    Document: {x: 1, y: 1, z: 1}
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: CRUDOpsOnUnshardedCollection
+  Type: CrudActor
+  Threads: 1  # We want to use 1 thread to avoid updates throwing WriteConflict errors.
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: *DocumentCount
+    Database: *Database
+    Collection: *UnshardedCollection
+    Operations:
+    - OperationMetricsName: UpdateOneWithoutShardKeyUnshardedCollection
+      OperationName: updateOne
+      OperationCommand:
+        Filter: {y: 1}
+        Update: {$inc: {y: 1}}
+        Options:
+          WriteConcern:
+            Level: majority
+            TimeoutMillis: 5000
+      OutcomeCounts:
+      - Filter: {y: 2}
+        Count: 1000
+  - Repeat: *DocumentCount
+    Database: *Database
+    Collection: *UnshardedCollection
+    Operations:
+    - OperationMetricsName: FindAndModifyOneWithoutShardKeyUnshardedCollection
+      OperationName: findOneAndUpdate
+      OperationCommand:
+        Filter: {y: 2}
+        Update: {$inc: {y: 1}}
+        Options:
+          WriteConcern:
+            Level: majority
+            TimeoutMillis: 5000
+        OutcomeCounts:
+        - Filter: {y: 3}
+          Count: 1000
+  - Repeat: *DocumentCount
+    Database: *Database
+    Collection: *UnshardedCollection
+    Operations:
+    - OperationMetricsName: DeleteWithoutShardKeyUnshardedCollection
+      OperationName: deleteOne
+      OperationCommand:
+        Filter: {z: 1}
+        Options:
+          WriteConcern:
+            Level: majority
+            TimeoutMillis: 5000
+      OutcomeCounts:
+      - Filter: {z: 1}
+        Count: 0
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - shard
```

Comment:
I'm not sure if these variants are all single shard variants. Can you check with server testing to make sure? I think also https://github.com/10gen/dsi/tree/master/configurations/mongodb_setup is where all the configurations are defined.